src/main/java/net/openhft/chronicle/wire/AbstractMethodWriterInvocationHandler.java

```diff
@@ -42,7 +42,7 @@ public abstract class AbstractMethodWriterInvocationHandler extends AbstractInvo
     private BiConsumer<Method, Object[]> handleInvoke;
     private boolean useMethodIds;
 
-    protected AbstractMethodWriterInvocationHandler(Class tClass) {
+    protected AbstractMethodWriterInvocationHandler(Class<?> tClass) {
```

Comment:
Because this class is in the public API, anyone can extend the class, and if such an extending class exists and overrides a raw parameter it may lead to a problem. I think we should do this change anyhow.